src/architecture/storage.md

```diff
@@ -1,301 +1,40 @@
 # Exonum Data Model
 
-This page describes Exonum **data storage** principles, from the database engine
-used (RocksDB), to the abstractions that are used in client
-applications.
+This page provides an overview of how Exonum works with the persistent data.
 
-1. [Exonum table types](#table-types) lists supported types of
-   data storage collections. Tables represent the highest abstraction level
-   for data storage
-2. [Low-level storage](#low-level-storage) explains how tables are persisted
-   using RocksDB
-3. [View layer](#view-layer) describes the wrapper over the DB engine
-   that ensures atomicity of blocks and transactions
-4. [List of system tables](#system-tables) contains tables
-   used directly by the Exonum core
-5. [Indexing](#indexing) gives an insight how indices over structured data
-   can be built in Exonum
+Exonum uses MerkleDB as a storage framework. MerkleDB is an object database. It
+represents a high-level wrapper over the key-value store.
 
-## Table Types
+Currently [RocksDB][rocks-db] is used for low-level key-value storage engine in
```

Comment:
Nit: I'd move this to the previous statement:

> [...] a high-level wrapper over the key-value store (RocksDB).

The present intro formatting seems too fragmented to me.

Reply:
done